index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Nabil's Blog</title>
```

Comment:
Don't forget to change the title of your pages - sometimes it can lead to embarassing situations. 😜